helm/cluster-cloud-director/values.yaml

```diff
@@ -208,9 +208,7 @@ cluster:
     provider: cloud-director
     resourcesApi:
       bastionResourceEnabled: false
-      cleanupHelmReleaseResourcesEnabled: false
-      clusterResourceEnabled: true
-      controlPlaneResourceEnabled: true
+      cleanupHelmReleaseResourcesEnabled: true
```

Comment:
if we enable this then we can drop the helmreleases cleanup hook from this chart